### PR TITLE
Add a simple README.md file

### PR DESCRIPTION
**Pull request by boh-prog**
Added a README file that was missing in repo

### CODE REVIEW

**Comment by perry-prog** [comment]
The read me is too short. Will like to have an example in it.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName,FirstName" into "Firstname LastName"
```
Comment (perry-prog) [inline README.md]: This Sentence is mission a period at the end.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Example
```
Comment (perry-prog) [inline README.md]: Please make this ## so that it's a smaller title

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Calling `rearrange_name("Turing, Alan")` will return `"Alan Turing"`
```
Comment (perry-prog) [inline README.md]: This is cool. But a couple of more example will be great!

**Comment by boh-prog** [comment]
All comments have been resolve. Please take another look.

**Pull request merged**